src/KeyboardInputSchemes.js

```diff
@@ -36,6 +36,8 @@ export type ActionName =
     | "selectLeft90"
     | "selectRight45"
     | "selectRight90"
+    | "selectRight180"
```

Comment:
We have removed turn right 180 action and don't need the "selectRight180" shortcut anymore.